### PR TITLE
Added exercises to Chapter 2

### PR DESCRIPTION
**Pull request by hjsuh94**
The links are valid as soon as the actual exercises PR comes through.

### CODE REVIEW

File: robot.html [hunk 1]
```diff
@@ -809,6 +809,32 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
...
+      <p> For this exercise you will investigate the effect of reflected inertia on the joint-space dynamics of the robot, and how it affects simple position control laws. You will work exclusively in <a href="https://colab.research.google.com/github/RussTedrake/manipulation/blob/master/exercises/robot/reflected_inertia.ipynb" target="_blank">this notebook</a>. You will be asked to complete the following steps. </p>
+
+      <ol type="a">
+        <li> Derive the first-order state-space dynamics $\dot{\bx} = f(\bx, \bu)$ of a simple pendulum with a motor and gearbox. 
+        </li>
+        <li> Compare the behavior of the direct-driven simple pendulum and the simple pendulum with a high-ratio gearbox, under the same position control law.</li>
+      </ol>
+    </exercise>
+
+    <exercise><h1>The Manipulation Station</h1>
```
Comment (RussTedrake) [inline robot.html:826]: How about "Input and Output Ports on the Manipulation Station"

File: robot.html [hunk 2]
```diff
@@ -809,6 +809,32 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
   appendix</a>.</p>
   
   </section>
+
+  <section><h1>Exercises</h1>
+
+    <exercise><h1>Role of Reflected Inertia</h1>
+
+      <p> For this exercise you will investigate the effect of reflected inertia on the joint-space dynamics of the robot, and how it affects simple position control laws. You will work exclusively in <a href="https://colab.research.google.com/github/RussTedrake/manipulation/blob/master/exercises/robot/reflected_inertia.ipynb" target="_blank">this notebook</a>. You will be asked to complete the following steps. </p>
```
Comment (RussTedrake) [inline robot.html:817]: should end in a colon, not period?

File: robot.html (hunk 1 above)
Comment (hjsuh94) [reply robot.html]: @RussTedrake changed in the new file! (sorry for the rebase)

File: robot.html (hunk 2 above)
Comment (hjsuh94) [reply robot.html]: @RussTedrake changed in the new file!

**Comment by hjsuh94** [comment]
I've also added Exercises section to index.html. (Or rather, found out it got magically added)

File: robot.html [hunk 3]
```diff
@@ -809,6 +809,32 @@ <h1><a href="index.html" style="text-decoration:none;">Robotic Manipulation</a><
...
+    </exercise>
+
+    <exercise><h1>Input and Output Ports on the Manipulation Station</h1>
+
+      <p> For this exercise you will investigate how a manipulation station is abstracted in Drake's system-level framework. You will work exclusively in <a href="https://colab.research.google.com/github/RussTedrake/manipulation/blob/master/exercises/robot/manipulation_station_io.ipynb" target="_blank">this notebook</a>. You will be asked to complete the following steps: </p>
+
+      <ol type="a">
+        <li> Query one of the output ports of the manipulation stations and compare the estimated values against the commanded values.
+        </li>
+        <li> Explain the discrepancy between the feed-forward torque and the commanded torque term in the manipulation station.</li>
```
Comment (RussTedrake) [inline robot.html:833]: i don't really like the descriptions here.  i feel like the important idea is just that we are learning to use the inputs and outputs of the manipulation station.  these details are not exactly right, and are not really the point of the exercise.

Comment (hjsuh94) [reply robot.html:833]: I see - I might have overdescribed it. What do you feel about:
- Probe into the output ports of the manipulation station and evaluate its content. 
- Explore the inner workings of the manipulation station by comparing the input feed-forward torque and the output commanded torque.

**Pull request merged**